content/influxdb/cloud/reference/internals/durability.md

```diff
@@ -1,9 +1,8 @@
 ---
 title: InfluxDB Cloud data durability
 description: >
-  InfluxDB Cloud ensures the durability of all stored data by replicating data across
-  multiple availability zones in a cloud region, automatically creating backups,
```

Comment:
This text is actually correct and doesn't need to be changed. We do replicate data across multiple availability zones (because AWS S3 does that for us). We don't run clusters in multiple AZs any more, but the data is replicated.